gas-station/typescript/src/sponsored_examples.ts

```diff
@@ -1,163 +1,240 @@
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
+
+// const walletClient = new WalletClient(WALLET_SERVICES_KEY);
+// const SENDER_ADDRESS = await walletClient.getWallet(INVISIBLE_WAL_ID);
+
+// 8. Sign the full transaction payload with the sender's key.
+// const keyClient = new KeyClient(WALLET_SERVICES_KEY);
+
+// const sessionToken = await keyClient.createSession(INVISIBLE_WAL_SECRET);
+// const senderSig = await walletClient.signTransactionBlock(
+//   INVISIBLE_WAL_ID,
+//   sessionToken,
+//   sponsoredResponse.txBytes
+// );
+
+// const WALLET_SERVICES_KEY = "{{walletAccessKey}}";
+// const INVISIBLE_WAL_SECRET = "{{walletSecret}}";
+// const INVISIBLE_WAL_ID = "{{walletId}}}";
+
+// -- Invisible Wallet: sponsor, sign, and execute in one method call -- //
+// import { 
+//   KeyClient, 
+//   WalletClient, 
+//   createSuiClient, 
+//   buildGaslessTransactionBytes 
+// } from "@shinami/clients";
+
+// // 1. Copy your Node and Wallet Services Testnet Api Key Values
+// const NODE_TESTNET_ACCESS_KEY = "sui_testnet_bb0ae4d25b7e51a98c8fcb267babc1b6";
+// const WALLET_SERVICES_ONLY_KEY = "3ccf23b29a99cb4d4c2974e590575f23";
+// const ALL_SERVICES_TESTNET_KEY = "sui_testnet_a683fdb22331e5a9ef146b4dc2f5cf67";
+
+// // 2. Set up your Node client
+// const nodeClient = createSuiClient(NODE_TESTNET_ACCESS_KEY);
+
+// // 3. Generate the TransactionKind for sponsorship as a Base64 encoded string
+// const gaslessPayloadBase64 = await buildGaslessTransactionBytes({
+//   sui: nodeClient,
+//   build: async (txb) => {
+//     txb.moveCall({
+//       target: "0xfa0e78030bd16672174c2d6cc4cd5d1d1423d03c28a74909b2a148eda8bcca16::clock::access",
+//       arguments: [txb.object('0x6')]
+//     });
+//   }
+// });
+
+// // 4. Set the Id and Secret values for a previously created Invisible Wallet
+// const INVISIBLE_WAL_SECRET = "akeuy32wekd";
+// const INVISIBLE_WAL_ID = "aaron";
+
+// // 5. Create a wallet and a key client. The wallet client needs an access key
+// //    with permissions for Wallet Services, Gas Sation, and Node Services
+// const walletClient = new WalletClient(ALL_SERVICES_TESTNET_KEY);
+// const keyClient = new KeyClient(WALLET_SERVICES_ONLY_KEY);
+
+// // 6. Generate a temporary session token 
+// const sessionToken = await keyClient.createSession(INVISIBLE_WAL_SECRET);
+
+// // 7. Set your gas budget
+// const GAS_BUDGET = 5_000_000;
+
+// // 8. Sign, sponsor, and execute in one call!
+// const signSponsorExecuteResponse = await walletClient.executeGaslessTransactionBlock(
+//   INVISIBLE_WAL_ID,
+//   sessionToken,
+//   gaslessPayloadBase64,
+//   GAS_BUDGET,
+//   { showEffects: true },
+//   "WaitForLocalExecution"
+// );
+
+// // 9. View the result
+// console.log(signSponsorExecuteResponse);
+
+
+// -- A different way to generate gasless transaction bytes as a base64 encoded string -- //
```

Comment:
Move this section to be aligned with its position in the tutorial

Reply:
fixed